src/olympia/devhub/views.py

```diff
@@ -882,6 +891,15 @@ def addons_section(request, addon_id, addon, section, editable=False):
             queryset=addon.addons_dependencies.all(), addon=addon,
             prefix='dependencies')
 
+    if section == 'technical':
+        try:
+            whiteboard = Whiteboard.objects.get(pk=addon.pk)
+        except Whiteboard.DoesNotExist:
+            whiteboard = Whiteboard(pk=addon.pk)
+
+        whiteboard_form = PublicWhiteboardForm(request.POST or None,
+                                               instance=whiteboard)
```

Comment:
Add a [`prefix="whiteboard"`](https://docs.djangoproject.com/en/1.11/ref/forms/api/#prefixes-for-forms) to avoid just calling the field name `public`, cause it's a little too generic and susceptible to clashes. (this will break tests above)